Oleksadr_Petrov/HW3/Task6.py

```diff
@@ -0,0 +1,6 @@
+names = ["Minodora Áurea", "Itoro Alban", "Gabriella Emil", "Sigmund Govinda", "Minodora Áurea", "Itoro Alban"]
+result = []
+for name in names:
+    if name not in result:
+        result.append(name)
```

Comment:
let it be. 
But try to solve it using fromkeys() function

Reply:
changes applied